src/app/services/evento.service.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138913)
```suggestion
import { Injectable } from "@angular/core";
```